devp2p/tests/test_slogging.py

```diff
@@ -1,7 +1,10 @@
 import logging
 import unittest
 from devp2p import slogging
-from StringIO import StringIO
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
nitpick

You this line works for both py2 and py3 so you can remove the try/catch and replace it with this single import.